python_ta/checkers/forbidden_import_checker.py

```diff
@@ -119,3 +129,13 @@ def get_allowed_local_files(self) -> list:
 def register(linter: PyLinter) -> None:
     """Required method to auto register this checker"""
     linter.register_checker(ForbiddenImportChecker(linter))
+
+
+def _get_full_import_names(modname: str, names: list) -> list:
```

Comment:
About type annotations: at the top of this file, add the line `from __future__ import annotations`, and then here you'll be able to use the more specific type annotations.